### ISSUE TITLE
Extend `newline-per-chained-call` rule to require every method call to be on a new line

### ISSUE THREAD

**Issue by adius** [eslint/eslint#9482]
Currently this is valid for `{ "ignoreChainWithDepth": 2 }`

```js
_.chain({})
  .map(foo)
  .filter(bar)
```
there should be flag to make it an error as well and require it to look like:

```js
_
  .chain({})
  .map(foo)
  .filter(bar)
```

**Comment by not-an-aardvark** [comment]
Thanks for your interest in improving ESLint. Unfortunately, it looks like this issue didn't get consensus from the team, so I'm closing it. We define [consensus](http://eslint.org/docs/maintainer-guide/issues.html#consensus) as having three :+1:s from team members, as well as a team member willing to champion the proposal. This is a high bar by design -- we can't realistically accept and maintain every feature request in the long term, so we only accept feature requests which are useful enough that there is consensus among the team that they're worth adding.

Since ESLint is pluggable and can load custom rules at runtime, the lack of consensus among the ESLint team doesn't need to be a blocker for you using this in your project, if you'd find it useful. It just means that you would need to [implement the rule](http://eslint.org/docs/developer-guide/working-with-rules) yourself, rather than using a bundled rule that is packaged with ESLint.
